AutowareAuto/src/common/autoware_auto_common/include/common/types.hpp

```diff
@@ -120,6 +120,29 @@ struct COMMON_PUBLIC PointXYZI
   }
 };
 
+#pragma pack(push,1)
+struct COMMON_PUBLIC PointXYZIRing
```

Comment:
Can we include a comment with a general description of `PointXYZIRing`?

Reply:
Comment added!